modules/videopress/class.videopress-xmlrpc.php

```diff
@@ -133,12 +133,6 @@ public function update_videopress_media_item( $request ) {
 		// update the meta to tell us that we're processing or complete
 		update_post_meta( $id, 'videopress_status', videopress_is_finished_processing( $id ) ? 'complete' : 'processing' );
 
-		// Get the attached file and if there isn't one, then let's update it with the one from the server.
-		$file = get_attached_file( $id );
-		if ( ! $file && is_string( $info['original'] ) ) {
-			videopress_download_video( $info['original'], $id );
```

Comment:
I left this for @georgestephanis, since he's a big proponent of downloaded videos. I'm not so sure about them, though. I'll delete as we can always pull this back out of source control later if needed.